sprints/sustainability/utils.py

```diff
@@ -0,0 +1,23 @@
+from typing import (
+    Dict,
+    List,
+)
+
+from sprints.dashboard.libs.jira import (
+    Account,
+)
+
+
+def split_accounts_into_categories(accounts: List[Account]) -> Dict[str, List[Account]]:
+    """
+    Converts `List[Account]` into the following format: `Dict[category: str, List[Account]]`.
+    """
+    result: Dict[str, List[Account]] = {}
+    for account in accounts:
+        try:
+            category = getattr(account, 'category').key
+        except AttributeError:
+            category = ''
```

Comment:
Will we handle this case in code? Ideally, all accounts should have a category, no?

Reply:
That's right. The only useful case could be debugging. I just thought it would be cleaner to do this with catching one exceptions instead of nesting `if`s for handling unexpected cases (no category, no key (if it's possible)).